src/mmw/apps/modeling/views.py

```diff
@@ -227,29 +227,35 @@ def _initiate_tr55_job_chain(model_input, job_id):
 @decorators.api_view(['GET'])
 @decorators.permission_classes((AllowAny, ))
 def boundary_layers(request, table_id=None, obj_id=None):
+    layer_list = settings.BOUNDARY_LAYERS
```

Comment:
Code looks good, but I'm not totally sure of what the arguments are and what the conditionals are checking for exactly. Consider adding a docstring and/or some comments.